package.json

```diff
@@ -1,20 +0,0 @@
-{
-  "name": "metomi-rose",
-  "private": true,
-  "scripts": {
-    "lint": "eslint -c .eslintrc.js sphinx/"
-  },
-  "dependencies": {
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^5.0.0"
```

Comment:
I'm not sure running `npx eslint` would have any way of knowing about these plugin dependencies anymore

Reply:
Would have to add a violation to one of the rules supplied by one of these plugins to see any difference

Reply:
That is what I tried yes.

Reply:
In the latest run

> ESLint couldn't find the config "standard" to extend from.